packages/reactivity/src/effect.ts

```diff
@@ -171,7 +171,7 @@ export function trigger(
   oldTarget?: Map<unknown, unknown> | Set<unknown>
 ) {
   const depsMap = targetMap.get(target)
-  if (!depsMap) {
+  if (!(depsMap?.size)) {
```

Comment:
You probably want to avoid using the null coalescing operator since it gets transpiled later

```suggestion
  if (!depsMap || !depsMap.size) {
```

Reply:
Thank you, I will rewrite it.